### PR TITLE
fix: [cp24]Saperate L0 and Mix trigger interval

### PR DESCRIPTION
**Pull request by XuanYang-cn**
See also: #37108
pr: #37190

- Add MixCompactionTriggerInterval, default 60s
- Add L0CompactionTriggerInterval, default 10s
- Export Single related compaction configs
- Raise SingleCompactionDeltaLogMaxSize from 2MB to 16MB

### CODE REVIEW

**Comment by XuanYang-cn** [comment]
rerun go-sdk

**Comment by XuanYang-cn** [comment]
rerun ut

File: pkg/util/paramtable/component_param.go [hunk 1]
```diff
@@ -3572,6 +3584,22 @@ During compaction, the size of segment # of rows is able to exceed segment max #
...
+	p.MixCompactionTriggerInterval.Init(base.mgr)
+
+	p.L0CompactionTriggerInterval = ParamItem{
+		Key:          "dataCoord.compaction.levelzero.triggerInterval",
+		Version:      "2.4.15",
+		Doc:          "The time interval in seconds for trigger L0 compaction, default as 10s",
+		DefaultValue: "10",
+	}
+	p.L0CompactionTriggerInterval.Init(base.mgr)
+
```
Comment (tedxu) [inline pkg/util/paramtable/component_param.go:3602]: Consider export the 2 params.

File: pkg/util/paramtable/component_param.go [hunk 2]
```diff
@@ -3572,6 +3584,22 @@ During compaction, the size of segment # of rows is able to exceed segment max #
...
+		Version:      "2.4.15",
+		Doc:          "The time interval in seconds for trigger mix compaction, default as 60s",
+		DefaultValue: "60",
+	}
+	p.MixCompactionTriggerInterval.Init(base.mgr)
+
+	p.L0CompactionTriggerInterval = ParamItem{
+		Key:          "dataCoord.compaction.levelzero.triggerInterval",
+		Version:      "2.4.15",
+		Doc:          "The time interval in seconds for trigger L0 compaction, default as 10s",
```
Comment (tedxu) [inline pkg/util/paramtable/component_param.go:3598]: Don't mention the default value in `Doc` field, we'll generate the default value according to `DefaultValue`.

File: internal/datacoord/compaction_trigger.go [hunk 3]
```diff
@@ -108,7 +108,7 @@ func newCompactionTrigger(
 }
 
 func (t *compactionTrigger) start() {
-	t.globalTrigger = time.NewTicker(Params.DataCoordCfg.GlobalCompactionInterval.GetAsDuration(time.Second))
+	t.globalTrigger = time.NewTicker(Params.DataCoordCfg.MixCompactionTriggerInterval.GetAsDuration(time.Second))
```
Comment (tedxu) [inline internal/datacoord/compaction_trigger.go:111]: Is `GlobalCompactionInterval` deprecated? If so, please note it in the `doc` field.

**Comment by XuanYang-cn** [comment]
rerun ut

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**